src/System.Net.Security/tests/FunctionalTests/System.Net.Security.Tests.csproj

```diff
@@ -40,6 +40,12 @@
     </Compile>
 
   </ItemGroup>
+  <ItemGroup Condition=" '$(TargetsUnix)' == 'true' "> 
+ 
+    <Compile Include="SslStreamStreamToStreamTest.cs" />   
+    <Compile Include="TestConfiguration.cs" />    
+       
+  </ItemGroup>
```

Comment:
other tests cannot be run in Unix as they depend upon system.net.socket dll for making socket connection (which is not available for sometime) so these are the few tescases  which can be run for the sanity check during PRs. This is a temporary fix.